mobile/examples/super_resolution/MAUI/MauiSuperResolution/OrtInferenceSession.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.ML.OnnxRuntime;
+using Microsoft.ML.OnnxRuntime.Tensors;
+using System.Diagnostics;
+
+namespace MauiSuperResolution
+{
+    public enum ExecutionProviders
+    {
+        CPU,   // CPU execution provider is always available by default
+        NNAPI, // NNAPI is available on Android
+        CoreML, // CoreML is available on iOS/macOS
+        XNNPACK // XNNPACK is available on ARM/ARM64 platforms and benefits 32-bit float models
+    }
+
+    // An inference session runs an ONNX model
+    class OrtInferenceSession
+    {
+        public OrtInferenceSession(ExecutionProviders provider = ExecutionProviders.CPU)
+        {
+            _sessionOptions = new SessionOptions();
+            switch (_executionProvider)
```

Comment:
```suggestion
            switch (provider)
```
maybe we don't need `_executionProvider`